### PR TITLE
Misc GATK updates

### PR DESCRIPTION
**Pull request by mlathara**
expose consolidate only option and tweak incremental import json update

### CODE REVIEW

File: example/java/TestGenomicsDBImporterWithMergedVCFHeader.java [hunk 1]
```diff
@@ -32,17 +32,27 @@
...
+    // similarly --consolidate-only. and they should be mututally exclusive
+    int numPartitions = 0, doConsolidate = -1;
     if (args[args.length-2].equals("--coalesce-multiple-contigs")) {
       numPartitions = Integer.parseInt(args[args.length-1]);
       args = Arrays.copyOf(args, args.length-2);
     }
+    else if (args[args.length-2].equals("--consolidate-only")) {
+      doConsolidate = Integer.parseInt(args[args.length-1]);
+      args = Arrays.copyOf(args, args.length-2);
+    }
```
Comment (kgururaj) [inline example/java/TestGenomicsDBImporterWithMergedVCFHeader.java:44]: Perhaps move these statements to inside the CommandLineImportConfig class and avoid position dependent if statements?

**Review by kgururaj — APPROVED** [review]
Minor comment

File: example/java/TestGenomicsDBImporterWithMergedVCFHeader.java (hunk 1 above)
Comment (mlathara) [reply example/java/TestGenomicsDBImporterWithMergedVCFHeader.java]: Done. Being lazy there would only lead to annoying test failures later...

**Review by nalinigans — APPROVED** [review]
Looks good.

**Comment by mlathara** [comment]
I just had to restart them a few times...some transient travis issue I guess 🤷

**Pull request merged**